src/fetch.js

```diff
@@ -51,6 +51,18 @@ const cleanAttributes = (attributes, currentSchema, schemas) => {
       return acc;
     }
 
+    // Changing the format in order to extract images from the richtext field
+    // NOTE: We could add an option to disable the extraction
+    if (attribute.type === 'richtext') {
+      return {
+        ...acc,
+        [attributeName]: {
+          [attributeName]: value,
```

Comment:
Maybe this doesn't make sense but Instead of having `attributeName` twice as the key could we not add some sort of specifier to one of the keys? 

```suggestion
        [`${attributeName}Data`]: {
          [attributeName]: value,
```

I don't know if it's possible but it might be a way to avoid some of that nesting we saw earlier like:

```javascript
body {
  body: {
  }
  localFiles: []
}
```
with 

```javascript
bodyData {
  body: {
  }
  localFiles: []
}
```

Reply:
What about:

```
body {
  data {
     childMarkdownRemark {
        html
    }
  }
  files {
      
   }
```

Reply:
In your last example, shouldn't the bodyData be the nested one?

But I agree, I think having the same name for the nested attributes can create some confusion. On the frontend, if you have a variable or prop named attributeName, you wouldn't know immediately if it's the parent or child one. And I like your proposal of attributeName+Data

Reply:
yeah you are right @remidej it should probably be the nested one. I like @soupette's idea too, if we can just put `data` that makes sense to me.

Reply:
Yes I think the last proposal (with `data`) might be the best. The properties are more predictable, so it makes it easier to create a rich text react component on the frontend

Reply:
👍